### ISSUE TITLE
Questions regarding the lr used for DiT-L

### ISSUE THREAD

**Issue by a-little-hoof** [LTH14/rcg#49]
Hi there! Thank you for providing the code — I really like your work!

I have a question about the learning rate you used in training DiT-L:

In the training script in the README, you set the learning rate (lr) to 1e-4, while the batch size is 2048. As a result, blr = lr*256/batch_size = 1.25e-5

However, in the paper, you mentioned that you follow the exact setting in _Deconstructing denoising diffusion models for
self-supervised learning_, which uses blr = 1e-4.

<img width="1126" height="284" alt="Image" src="https://github.com/user-attachments/assets/bf3f7d8b-55e7-4359-bd03-5f3f7f8698fe" /> 

I wonder whether this difference will have an impact on the training. We are doing an apple-to-apple comparison with other baselines.

I really appreciate your help.

**Comment by LTH14** [comment]
I think it should be a typo -- it should be blr
